client/src/pages/Login.js

```diff
@@ -527,7 +527,7 @@ const Login = ({ isLoginForm, forgotPassword }) => {
             <WhiteSpace />
             {!forgotPassword && (
               <SectionDiv className="text-center">
```

Comment:
Above here, there is line 523, it says Back to Login page, change it to back to sign in page.

Reply:
updated